components/function-controller/README.md

```diff
@@ -18,13 +18,39 @@ This Knative-based serverless implementation defines and handles the Function Cu
 ### Run locally
 Follow these steps to run the Knative Function controller locally:
 
-1. Modify the `config/config.yaml` file to include your base64-encoded `gcr.io` or `docker.io` credentials. 
-2. Update the Docker registry value to your `docker.io` username.
+1. Create a ServiceAccount to enable knativeBuild Docker builds.
 
-3. Apply the configuration:
+To run a build in a Namespace, create ServiceAccounts with linked Docker repository credentials for each Namespace that
+ will be used for the function-controller.
 
 ```bash
-kubectl apply -f config/config.yaml
+#set your namespace e.g. default
+export NAMESPACE=<NAMESPACE>
+#set your registry e.g. https://gcr.io
+export REGISTRY=<YOURREGISTRY>
+echo -n 'Username:' ; read username
+echo -n 'Password:' ; read password
+cat <<EOF | sed s/hihihih/hohohoho/
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: function-controller-build
```

Comment:
`function-controller-build` is not used anywhere. Is it supposed to replace `serverless-controller` ?

Reply:
the name of the controller is still `function-controller` only the sub-domain changed

Reply:
fixed

Reply:
As discussed: should be the same name as in `config.yaml`